tests/fuzz/fuzz_common.h

```diff
@@ -0,0 +1,62 @@
+//
+// Created by profanter on 18.07.17.
+// Copyright (c) 2017 fortiss GmbH. All rights reserved.
+//
+
+#ifndef OPEN62541_FUZZ_COMMON_H_H
```

Comment:
This file does the same as testing_networklayer.c/.h.
Is there a reason to duplicate?